olwebsite-app/src/components/SearchBox/SearchBox.jsx

```diff
@@ -0,0 +1,49 @@
+import React, {useState} from "react";
```

Comment:
should this file be a js file or jsx file?

Reply:
eh im for using jsx for files the contain jsx

Reply:
I prefer jsx as well; it provides more convenience. Can we add that to the Coding Standards for components?

Reply:
I made a proposal for it